guessing_game.py

```diff
@@ -1,15 +1,28 @@
 
 import random
 import math
+import unittest
 
 gameChoice = "Y"
-numberOfGuesses = 8
 welcomeText=['\n\n--------------------Guessing game--------------------\n\n',
              'Welcome to the guessing game!',
              'You have 8 tries to guess the number.',
              'After 5 wrong tries, you will be provided with a hint.',
              'Good Luck!']
 
+class TestGuessingGame(unittest.TestCase):
+
+    def test_calcScore(self):
```

Comment:
Maybe add a docstring to elaborate whats happening in the test. https://www.programiz.com/python-programming/docstrings

This practice allows developers to understand why this test was written. When working wiht a big chunk of code, itll allow you to remember why yourself had written code too, because often times you would forget when working with thousands of files.